active/0000-firefox-in-ci.md

```diff
@@ -0,0 +1,59 @@
+- Start Date: 2016-12-04
+- RFC PR: 
+- Ember CLI Issue: 
+
+# Summary
+
+Replace PhantomJS with Firefox as the default browser for continuous integration testing.
+
+# Motivation
+
+We want to provide the best possible out-of-the-box continuous integration testing experience for Ember apps. Today that means shipping with configurations for testem and TravisCI. Those configurations use PhantomJS.
+
+But PhantomJS is a weird environment. Users must often fix Phantom-specific browser bugs, which is wasted effort since real users never run your app in Phantom. And "how to debug in Phantom" is an entire extra skill people are forced to learn.
+
+A user-targeted, standards-compliant, modern browser makes a better default choice. Firefox is a good candidate because it's 100% open source, well-supported by Testem on all major operating system, and built-in to TravisCI. Debugging in Firefox has a dramatically nicer learner curve than PhantomJS.
+
+# Detailed design
+
+This is a proposed change to the blueprints for new apps and addons. Existing apps and addons would only be affected when they re-run `ember init` as part of an upgrade and choose to take the updated configuration.
+
+## Changes in testem.js
+
+Replace `PhantomJS` with `Firefox`.
+
+## Changes in travis.yml
+
+Add the following new section to start up a virtual display:
+
+```
+before_script:
+  - export DISPLAY=:99; sh -e /etc/init.d/xvfb start; sleep 3
```

Comment:
@ef4 i think it might be nice to split this command over multiple separate steps:

* if one fails, the next doesn't happen  `;` without `&&` will cascade
* easier to quickly read/understand
* we can document the steps (such as, the sleep 3)

---  

Is there a way to avoid the sleep 3, feels a tad racy?